### ISSUE TITLE
default transparent background not suitable for high-contrast display settings

### ISSUE THREAD

**Issue by mhettema** [lrsjng/jquery-qrcode#47]
Hi, we recently found out that using the default transparent background might render certain A11Y display settings unusable. For instance with 'inverted colors' both the background of the webpage and the pixels of the QR code are then black... Not very A11Y friendly.

Might not be possible to change the default, but maybe warning in documentation?

**Comment by lrsjng** [comment]
Changed to `'#fff'` in the upcoming release.
